types/styled-components/index.d.ts

```diff
@@ -330,7 +330,9 @@ export type ThemedCssFunction<T extends object> = BaseThemedCssFunction<
 >;
 
 // Helper type operators
-type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+type Omit<T, K extends keyof any> = T extends any
```

Comment:
I don't really understand why you need to change to this.
The construction doesn't make much sense to me.
`T` always extends `any`, and why the constraints get relaxed?

Reply:
I just got this Omit from @material-ui/core, because current Omit doesn't work with this library

Reply:
Here or in styled-components repo? Should I close this pull request?

Reply:
Here, in DT, please.